language-design.md

```diff
@@ -17,7 +17,41 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+Quote 1: "Why shouldn’t those 'interfaces' be as humanely designed as the ones we tap and swipe?" [Pavlus, 2012].
+
+Here Pavlus asks why programming languages' interfaces are not as "humanely
+designed" as software's graphical interfaces are. This quote stood out to us
+because Pavlus is trying to make a point, but actually ends up asking a
+question that is very easy to answer! One answer is that in general, software
+(programming languages included) is designed towards its users. The software
+Pavlus is referring to is probably targeted at a much broader audience, so it
+needs to be easier. Also, we take issue with Pavlus implying that programming
+is comparable to tapping and swiping. Is Pavlus claiming that programming
+should be as easy as Angry Birds? I doubt that would be a productive language.
```

Comment:
I also agree with Adam here - be careful about making a claim that ease of use is inversely proportional to expressiveness or power of the toolset. Of course programming  and angry birds are very different, but what makes angry birds special perhaps is that it is easy to play, intuitive to pick up on how it works- and if programming was the same way, that might not be a bad thing. 

Also, a minimal tool set might not be a bad thing, even for a general language. As long as you can do everything you  are supposed to be able to do, not giving excess tools might even be a good thing.